database/sql/V056__remove_surplus_fee.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE orders 
+  DROP COLUMN surplus_fee,
+  DROP COLUMN surplus_fee_timestamp;
```

Comment:
Please remember to update the `database/README.md`
Also I think having a small comment for each DB migration explaining why it's necessary is kind of nice.

Reply:
Added, thanks. I always forget to update :)